docs/cosmos-based/parser/config/config.md

```diff
@@ -169,13 +169,19 @@ node:
             address: http://localhost:9090
             insecure: true
 
-# node type : local
+# node type : local, BDjuno is set to read from the chain's database of a static node
 node:
     type: local
     config:
         home: path/to/.desmos
 ```
 
+:::caution Node Type  
+If you encounter the error `ERROR: failed to start client: failed to initialize database: resource temporarily unavailable`
+, you might want to make sure the `node.type` is `remote` which tells BDjuno to read from a running node, instead of the static chain's DB.
```

Comment:
```suggestion
, you might want to make sure the `node.type` is set to`remote` which tells BDjuno to read from a running node, instead of the static chain's DB.
```